Makefile

```
@@ -349,3 +349,27 @@ integration-test:
 	else\
 		$(GINKGO) -v $(TEST_ARGS) --fail-fast --focus-file=$(TEST_FILE) test/integration;\
 	fi
+
+#hosted
+
+.PHONY: bootstrap-hosted
+bootstrap-hosted: clone-hosted install-hosted copy-config install-crds install-resources kind-deploy-policy-framework kind-deploy-policy-controllers
+
+.PHONY: install-hosted
+install-hosted: 
+	@cd ./governance-policy-addon-controller && KIND_VERSION=$(KIND_VERSION) HOSTED_MODE=true ./build/manage-clusters.sh
+
+.PHONY: 
+copy-config:
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig ./kubeconfig_hub
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl1.kubeconfig-internal ./kubeconfig_hub_internal
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig ./kubeconfig_managed
+	@cp ./governance-policy-addon-controller/policy-addon-ctrl2.kubeconfig-internal ./kubeconfig_managed_internal
+
+.PHONY: clone-hosted
+clone-hosted:
+	-git clone --depth=1 -b $(RELEASE_BRANCH) https://github.com/stolostron/governance-policy-addon-controller.git
```

Optional: Similar to [targets in the addon-controller](https://github.com/stolostron/governance-policy-addon-controller/blob/9a2988e4a251ffff48cf23237014fce4694a4103/Makefile#L218-L221), this target could be: `$(ADDON_CONTROLLER)`, and then you can use that variable for the path throughout as well as not have to ignore when the clone fails (because it won't run at all since the directory already exists).

If you go this route, I'd probably also just make it a prerequisite for `install-hosted` so that that target can run successfully by itself.

```suggestion
ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller
$(ADDON_CONTROLLER):
	git clone --depth=1 -b $(RELEASE_BRANCH) https://github.com/stolostron/governance-policy-addon-controller.git
```

---

oh good I didn't know this Thanks!